docs/contributing/testing.md

```diff
@@ -41,10 +41,11 @@ If you've created a new component, create a new `src/govuk/<COMPONENT>/<COMPONEN
 
 You should test that your contribution works:
 
-- in [recommended browsers](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in)
-- with [recommended assistive technologies](https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies#what-to-test)
 - in [Internet Explorer 8](https://frontend.design-system.service.gov.uk/supporting-ie8/), 9 and 10 - components do not need to look perfect
```

Comment:
Do users need to test in IE11?

Reply:
That's covered by the recommended browsers in the service standard (at least for now)

https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in

Reply:
Thanks @lfdebrux!